### ISSUE TITLE
Class Limits

### ISSUE THREAD

**Issue by Kirby859** [FlansMods/FlansMod#123]
I've been working on a content pack for a few hours, and I thought it would be neat to have limits on classes. Let's say I'm making a WWII themed content pack. Obviously most people would be riflemen, as they are the most common units. But in Minecraft, what's stopping everyone from all picking sniper? A simple solution would be class limits. This could be a simple addition to the class selection interface. Next to the name of the class, there could be a x/y counter that shows how many people have taken a class, and how many people the class can hold. An example can be found in Red Orchestra 2:  Heroes of Stalingrad:

http://cloud.steampowered.com/ugc/542907315069766517/2F420E02FCE9D830440E74E87880D8B94B1BB227/

So, if I had an Assault class, I might set the maximum to 5. The class name will show as Assault 2/5 if two people are on Assault.

tl;dr Give the option to set limits on the amount of players on classes.
